logstash-input-webfetcher/src/main/java/eu/wajja/web/fetcher/FetcherJob.java

```diff
@@ -97,19 +102,27 @@
 	private String crawlerUserAgent;
 	private String crawlerReferer;
 	private Long maxPagesCount = 0l;
+	private Set encounteredExcludedLinks;
+	private Long excludedDataPagesCount = 0l;

```

Comment:
do we even need this ? of we send the data for analytics we can get the total inside kibana. (group by threadid)

Reply:
Kibana is not reliable enough.